NPE when adding local images to legacy editor

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:325)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
	at java.lang.Thread.run(Thread.java:761)
 Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equalsIgnoreCase(java.lang.String)' on a null object reference
	at org.wordpress.android.util.ImageUtils.resizeImageAndWriteToStream(ImageUtils.java:498)
	at org.wordpress.android.util.ImageUtils.createThumbnailFromUri(ImageUtils.java:625)
	at org.wordpress.android.util.ImageUtils.getWPImageSpanThumbnailFromFilePath(ImageUtils.java:359)
	at org.wordpress.android.editor.LegacyEditorFragment.createWPEditImageSpanLocal(LegacyEditorFragment.java:479)
	at org.wordpress.android.editor.LegacyEditorFragment.createWPEditImageSpan(LegacyEditorFragment.java:506)
	at org.wordpress.android.editor.LegacyEditorFragment.access$1800(LegacyEditorFragment.java:74)
	at org.wordpress.android.editor.LegacyEditorFragment$AddMediaFileTask.doInBackground(LegacyEditorFragment.java:1054)
	at org.wordpress.android.editor.LegacyEditorFragment$AddMediaFileTask.doInBackground(LegacyEditorFragment.java:1036)
```

This is only reproducible in current `develop`, after `alpha-36`, and only with the legacy editor (not visual or Aztec).

--- a/libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java
+++ b/libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java
@@ -495,7 +495,7 @@ public class ImageUtils {
         }
 
         Bitmap.CompressFormat fmt;
-        if (fileExtension.equalsIgnoreCase("png")) {
+        if (fileExtension != null && fileExtension.equalsIgnoreCase("png")) {
             fmt = Bitmap.CompressFormat.PNG;
         } else {
             fmt = Bitmap.CompressFormat.JPEG;

